doc/source/getting_started/installing.rst

```diff
@@ -194,7 +206,7 @@ TL;DR
 Installing a Python interpreter
 ===============================
 Open a terminal and type ``python`` + Enter.
-If a Python prompt appears, you already have an interpreter installed and can skip this step (exit by pressing Ctrl+D).
+If a Python prompt appears, you already have an interpreter installed and can skip this step (exit by pressing Ctrl+D or running ``exit()``).
```

Comment:
dislike "a or b" for beginner level instructions, simply give them one option (here, id go for `exit()`)